sidebars.js

```diff
@@ -33,6 +33,11 @@ const sidebars = {
       label: 'Network',
       href: 'https://github.com/containers/podman/blob/main/docs/tutorials/basic_networking.md',
     },
+    {
+      type: 'link',
+      label: 'Podman Python',
+      href: 'https://podman-py.readthedocs.io/en/latest/index.html',
+    },
```

Comment:
@umohnani8  Is there Any way to merge this with the Podman Read the Docs site?  That's what I was hoping for at the onset.

Reply:
I don't think there is an easy way to do that, especially given since the docs for podman and podman-py are built quite differently and they are 2 different repos. The podman-py docs have a link to the podman docs and now the podman docs also have a link to the podman-py docs. I think this should be good enough for users when looking for documentation for both.